app/views/step_by_step_pages/show.html.erb

```diff
@@ -60,6 +58,10 @@
                   <%= link_to("Edit", edit_step_by_step_page_step_path(@step_by_step_page.id, step.id), class: "gem-c-button govuk-button gem-c-button--inline") %>
                   <%= link_to("Delete", step_by_step_page_step_path(@step_by_step_page.id, step.id), method: 'delete', data: { confirm: "Are you sure?" }, class: "gem-c-button govuk-button govuk-button--warning gem-c-button--inline") %>
                 </td>
+              <% else %>
+                <td class="govuk-table__cell govuk-table__cell--numeric">
+                  <%= link_to("View", edit_step_by_step_page_step_path(@step_by_step_page.id, step.id), class: "gem-c-button govuk-button gem-c-button--inline") %>
```

Comment:
As part of the scheduling work we decided not to allow users to visit the "Edit step" page, so this "View" link isn't needed.

Reply:
Ah interesting, because they can still view the step content via the step by step preview?

Because if we don't provide them with a 'View' link, they'll have no way of looking at the step markup (URLs etc). But I suppose that's not an issue so long as they can still see the output.

Reply:
Have removed 👍